jms/src/main/java/com/sun/ts/tests/jms/core/bytesMsgQueue/BytesMsgQueueTestsJspTest.java

```diff
@@ -75,27 +75,27 @@ public static EnterpriseArchive createDeploymentVehicle(@ArquillianResource Test
             com.sun.ts.tests.common.vehicle.VehicleClient.class
             );
             // The web.xml descriptor
-            URL warResURL = BytesMsgQueueTests.class.getResource("/com/sun/ts/tests/common/vehicle/jsp/jsp_vehicle_web.xml");
+            URL warResURL = BytesMsgQueueTests.class.getResource("/com/sun/ts/tests/jms/core/bytesMsgQueue/jsp_vehicle_web.xml");
```

Comment:
Hi @starksm64 , It seems like the web.xml, sun-web.xml locations need corrections for most of the tests here to pass with Glassfish. Can you please review if this looks ok .

Reply:
Yes, the local version of the descriptor should be preferred.